addons/common/fnc_headDir.sqf

```diff
@@ -54,49 +54,35 @@ Author:
 #include "script_component.hpp"
 SCRIPT(headDir);
 
-private["_azimuth", "_angle", "_dif", "_infov", "_object", "_threed", "_do"];
-
 params ["_unit"];
-_object = param [1,_unit];
-_threed = false;
-
-_do = (typeName _object != typeName _unit);
-if (!_do) then {
-    _do = (_object != _unit);
-};
+private _object = param [1, _unit];
+private _isExternalCam = false;
 
-if (_do) then {
-    _angle = [(_unit call CBA_fnc_getPos),(_object call CBA_fnc_getPos)] call BIS_fnc_dirTo;
-} else {
-    _angle = 0;
-};
-if (_unit != player) then
-{
-    _azimuth = getdir _unit;
+private _azimuth = 0;
+if (_unit != player) then {

```

Comment:
maybe replace this `player` with `call CBA_fnc_currentUnit`. That way the function wouldn't bug out when remote controlling as zeus.